starter-code/css/main.css

```diff
@@ -7,5 +7,57 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
-  background: linear-gradient(to bottom, #1c5380 50%, white 50%);  
+  background: linear-gradient(to bottom, #1c5380 40%, white 40%);
+  background-repeat: no-repeat;
+}
+
+.left-col {
+  margin: 100px auto auto 100px;
+  float: left;
+  width: 30%;
+}
+
+.right-col {
+  margin-top: 100px;
+  float: left;
+  width: 60%;
+}
+
+.brand {
+  size: 100px;
+  float: left;
+}
+
+.home {
```

Comment:
To get your home home button a bit closer to the example, try something like this:

 home {
float: right;
  margin-top: 20px;
  background: linear-gradient(#6c90ad, #4a7598);
  border: 1px solid #1c5380;
  border-radius: 2px;
  padding: 5px 7px;
  color: white;
}